jaxopt/_src/base.py

```diff
@@ -378,3 +378,8 @@ def run(self,
 
     return super()._run(init_stepsize, params, value, grad, descent_direction,
                         *args, **kwargs)
+
+
+class Initialization(NamedTuple):
+  params: Any
+  state: Any
```

Comment:
This is exactly the same as `base.OptStep` so I think we should use it instead of introducing `Initialization`. So basically your change allows to resume optimization from any given (point, state) tuple with `run`, i.e., without a manual for loop. CC @fabianp @fllinares

Reply:
It seems indeed a good idea to reuse OptStep here

Reply:
Agreed, do you think then it can be made into a more general change rather than focused on LBFGS?
Anyway updating now with OptStep